src/ctia/domain/entities/actor.clj

```diff
@@ -0,0 +1,13 @@
+(ns ctia.domain.entities.actor
+  (:require
+    [ctia.properties :refer [get-http-show]]
+    [ctim.domain.id :as id]))
+
+(def short-id->long-id
+  (id/factory:short-id->long-id :actor get-http-show))
+
+(defn with-long-id [{id :id, :as entity}]
+  (assoc entity :id (short-id->long-id id)))
+
+(defn page-with-long-id [{data :data, :as m}]
+  (assoc m :data (map with-long-id data)))
```

Comment:
My first inclination here is to use `update-in` rather than `assoc`, because it's more descriptive of what's happening, and there are fewer things going on. 

`(defn page-with-long-id [{data :data, :as m}]
   (update-in m [:data] (partial map with-long-id))`